math/asl/Portfile

```diff
@@ -0,0 +1,40 @@
+# -*- coding: utf-8; mode: tcl; tab-width: 4; indent-tabs-mode: nil; c-basic-offset: 4 -*- vim:fenc=utf-8:ft=tcl:et:sw=4:ts=4:sts=4
+
+PortSystem                      1.0
+PortGroup                       cmake 1.1
+PortGroup                       github 1.0
+PortGroup                       muniversal 1.1
+
+github.setup                    ampl asl 94efb9e220c963bddee3d4500c28e86eb560f858
+version                         20230818
+revision                        0
+
+categories                      math
+maintainers                     {@catap korins.ky:kirill} openmaintainer
+
+# It is similar for so-called CWI license which used for Python between 0.9.0 and 1.2
+license                         Unknown
```

Comment:
Kirill, is the license Permissive-ish, such that we can redistribute binaries? Or should we avoid distribution, just-in-case?

CC: @jmroot

Reply:
@mascguy as far as I see we can redistirbute it without any reason.

See the license here: https://github.com/ampl/asl/blob/master/src/solvers/asl.h#L1-L17

Reply:
A lot of the files don't have any license header, and there's no blanket license specified anywhere, which makes things questionable at best. https://github.com/ampl/asl/issues/7

Many of the files that do have a license are an MIT variant, but you'd have to go through all of them to make sure there aren't any other licenses in there.

Reply:
> A lot of the files don't have any license header, and there's no blanket license specified anywhere, which makes things questionable at best. [ampl/asl#7](https://github.com/ampl/asl/issues/7)
> 
> Many of the files that do have a license are an MIT variant, but you'd have to go through all of them to make sure there aren't any other licenses in there.

So would `Restrictive` be the safest at this point? Or does `Unknown` provide the same net effect?

Reply:
To be on safe side I've added `Restrictive` and keep `Unknown`.